requirements/base.txt

```diff
@@ -1,9 +1,10 @@
 astunparse
 pandas
+pyarrow
 numpy
 numba
-dask
-distributed>=2022.11.1
+dask[dataframe]>=2024.12.1
+distributed>=2024.12.1
```

Comment:
```suggestion
dask[dataframe]>=2025.2.0
distributed>=2025.2.0
```
does the most recent release (or `2025.1.0`) cause issues?

Reply:
Yeah, This is because we pin it for rapids here: 
https://github.com/rapidsai/rapids-dask-dependency/pull/77